fairlearn/metrics/__init__.py

```diff
@@ -18,81 +18,71 @@
 for each subgroup identified in ``group_membership``.
 """
 
-from ._extra_metrics import balanced_root_mean_squared_error, fallout_rate  # noqa: F401
-from ._extra_metrics import mean_overprediction, mean_prediction  # noqa: F401
-from ._extra_metrics import mean_underprediction, miss_rate  # noqa: F401
-from ._extra_metrics import selection_rate, specificity_score  # noqa: F401
-from ._extra_metrics import group_fallout_rate  # noqa: F401
-from ._extra_metrics import group_miss_rate  # noqa: F401
-from ._extra_metrics import group_specificity_score  # noqa: F401
-from ._extra_metrics import group_balanced_root_mean_squared_error  # noqa: F401
-from ._extra_metrics import group_mean_overprediction  # noqa: F401
-from ._extra_metrics import group_mean_prediction  # noqa: F401
-from ._extra_metrics import group_mean_underprediction  # noqa: F401
-
-from ._selection_rate import group_selection_rate  # noqa: F401
-
-from ._skm_wrappers import group_accuracy_score, group_confusion_matrix  # noqa: F401
-from ._skm_wrappers import group_precision_score, group_recall_score  # noqa: F401
-from ._skm_wrappers import group_roc_auc_score, group_zero_one_loss  # noqa: F401
-from ._skm_wrappers import group_max_error  # noqa: F401
-from ._skm_wrappers import group_mean_absolute_error  # noqa: F401
-from ._skm_wrappers import group_mean_squared_error  # noqa: F401
-from ._skm_wrappers import group_mean_squared_log_error  # noqa: F401
-from ._skm_wrappers import group_median_absolute_error  # noqa: F401
-from ._skm_wrappers import group_root_mean_squared_error  # noqa: F401
-from ._skm_wrappers import group_r2_score  # noqa: F401
-
-from ._metrics_engine import make_group_metric, group_summary  # noqa: F401
-from ._metrics_engine import group_min_from_summary, group_max_from_summary  # noqa: F401
-from ._metrics_engine import difference_from_summary, ratio_from_summary  # noqa: F401
-
-# -------------------------------------------
+from ._extra_metrics import (  # noqa: F401
+    true_positive_rate,
+    true_negative_rate,
+    false_positive_rate,
+    false_negative_rate,
+    balanced_root_mean_squared_error,
+    mean_overprediction,
+    mean_prediction,
+    mean_underprediction,
+    selection_rate)
+
+from ._metrics_engine import (  # noqa: F401
+    make_metric_group_summary, group_summary,
+    make_derived_metric,
+    group_min_from_summary, group_max_from_summary,
+    difference_from_summary, ratio_from_summary,
+    _metric_group_summary_dict, _derived_metric_dict)
+
+from ._disparities import (  # noqa: F401
+    demographic_parity_difference,
```

Comment:
Something (slightly off-topic) which came out of an AzureML-related review.... is 'demographic parity' a good name, given that different people might understand different things by 'demographic' (much the same way as we steer away from 'bias' in our writing?

We did explain that it's a term used in the literature, but it's something to consider. A more neutral term would be 'selection rate parity' I believe.

Reply:
Discussed separately. This has been talked about extensively, and 'demographic parity' found to be an appropriate term.